docs/readme/project_setup.md

```diff
@@ -1,14 +1,43 @@
 # Project Setup
 
+## Installation Prerequisites
+
+### JDK Versions
+
+Running Data Prepper requires JDK 8 and above.
+
+Running the integration tests requires JDK 14 or 15.
```

Comment:
nit - JDK 14 and above? (e.g. will 16 work?)

Reply:
@wrijeff , We need to update to Gradle 7 before we can build with JDK 16.